README.md

```diff
@@ -704,6 +704,25 @@ Only Odoo's:
 
     docker-compose run --rm odoo odoo -u addon1,addon2 --stop-after-init
 
+##### Update changed addons only
+
+Add `module_auto_update` from https://github.com/OCA/server-tools to your
+installation following the standard methods of `repos.yaml` + `addons.yaml`.
+
+Now we will install the addon:
+
+    docker-compose up -d
+    docker-compose run --rm odoo --stop-after-init -u base
+    docker-compose run --rm odoo --stop-after-init -i module_auto_update
+    docker-compose restart odoo
+
+It will automatically update addons that got updated every night.
+To force that automatic update now in a separate container:
+
+    docker-compose up -d
+    docker-compose run --rm odoo autoupdate
+    docker-compose restart odoo
```

Comment:
It's also worth noting that now that our strategy does use the `module_auto_update`, we explicitly disable the cron. 

Might be worth mentioning that in the instructions - when we tested it, the database would sometimes deadlock (in the event of a lot of updates). The deadlocks would cause our health checks to restart the containers, which would then quadruple down on the amount of Odoo instances trying to update the same database. It would have been a real mess had it not been in UAT.